Package/Assets/GDK-Tools/Source/Utilities/GdkUtilities.cs

```diff
@@ -17,7 +17,7 @@ public static string GdkToolsPath
             {
                 if (!File.Exists(_gdkToolsPath))
                 {
-                    _gdkToolsPath = Path.Combine(RegUtil.GetRegKey(RegUtil.HKEY_LOCAL_MACHINE, @"SOFTWARE\WOW6432Node\Microsoft\GDK", "InstallPath"), "bin");
+                    _gdkToolsPath = Path.Combine(RegUtil.GetRegKey(RegUtil.HKEY_LOCAL_MACHINE, @"SOFTWARE\WOW6432Node\Microsoft\GDK", "GRDKInstallPath"), "bin");
```

Comment:
Would be awesome if RegUtil.GetRegKey() didn't hard code the 64-bit view so we could avoid hardcoding WOW6432Node as a literal. Perhaps a future improvement if you don't want to get more involved with changes at this time.